nf_core/pipeline-template/{{cookiecutter.name_noslash}}/nextflow_schema.json

```diff
@@ -5,195 +5,237 @@
+            },
+            "fa_icon": "fas fa-university"
         },
-        "Maximum job request limits": {
+        "Max job request options": {
             "type": "object",
-            "description": "Limit the maximum computational requirements that a single job can request",
-            "default": "",
             "properties": {
                 "max_cpus": {
                     "type": "integer",
-                    "description": "Maximum number of CPUs that can be requested  for any single job",
+                    "description": "Maximum number of CPUs that can be requested  for any single job.",
                     "default": 16,
                     "fa_icon": "fas fa-microchip",
-                    "hidden": true
+                    "hidden": true,
+                    "help_text": "Use to set an upper-limit for the CPU requirement for each process. Should be an integer e.g. `--max_cpus 1`"
                 },
                 "max_memory": {
                     "type": "string",
-                    "description": "Maximum amount of memory that can be requested for any single job",
-                    "default": "128 GB",
+                    "description": "Maximum amount of memory that can be requested for any single job.",
+                    "default": "128.GB",
                     "fa_icon": "fas fa-memory",
-                    "hidden": true
+                    "hidden": true,
+                    "help_text": "Use to set an upper-limit for the memory requirement for each process. Should be a string in the format integer-unit e.g. `--max_memory '8.GB'`"
                 },
                 "max_time": {
                     "type": "string",
-                    "description": "Maximum amount of time that can be requested for any single job",
-                    "default": "10d",
+                    "description": "Maximum amount of time that can be requested for any single job.",
+                    "default": "240.h",
                     "fa_icon": "far fa-clock",
-                    "hidden": true
+                    "hidden": true,
+                    "help_text": "Use to set an upper-limit for the time requirement for each process. Should be a string in the format integer-unit e.g. `--max_time '2.h'`"
                 }
-            }
+            },
+            "fa_icon": "fab fa-acquisitions-incorporated"
         },
-        "Institutional config params": {
+        "Pipeline template options": {
             "type": "object",
-            "description": "Params used by nf-core/configs",
-            "default": "",
             "properties": {
-                "custom_config_version": {
+                "help": {
+                    "type": "boolean",
+                    "description": "Display help text.",
+                    "hidden": true,
+                    "fa_icon": "fas fa-question-circle",
+                    "default": false
+                },
+                "outdir": {
```

Comment:
I think this might make more sense near the start, as it's one of the most commonly changed values..?

Reply:
But that would mean I would have to move the whole group to the top? Its currently the 2nd in that group after `help`.

Reply:
Or I just move the `outdir` parameter into another group?

Reply:
Just the `outdir` parameter I was thinking, as it's such a commonly set parameter (much more so than the others in this group).

Reply:
Yep. That makes sense. Moved `outdir` and `email` and renamed that section to `Input/output options`
See https://github.com/nf-core/tools/pull/593/commits/39753feeb1353ce280cdb86f682c6acbe09e3b60